mopidy/file/library.py

```diff
@@ -82,7 +82,14 @@ def browse(self, uri):
             elif os.path.isfile(child_path):
                 result.append(models.Ref.track(name=name, uri=uri))
 
-        result.sort(key=operator.attrgetter('name'))
+        sort_funcs = [operator.attrgetter('name'),
+                      lambda item: 0 if item.type == models.Ref.DIRECTORY
+                      else 1]
+        logger.debug('sort_funcs %s', sort_funcs)
+
+        for key_func in sort_funcs:
```

Comment:
Doh, thanks for explaining that. Should have kept my original wording with "Doing it like this I think..." :-)